controller.py

```diff
@@ -550,6 +561,12 @@ def process_arguments(args):
         )
         exit(1)
 
+    if args.goldenrun_only:
+        parguments["goldenrun_only"] = True
+        parguments["goldenrun"] = True
+    else:
+        parguments["goldenrun_only"] = False
+
```

Comment:
As args.goldenrun_only is configured with `store_true` it will, if not set, default to `false`.
Hence, it could be a bit shortened:

```
parguments["goldenrun_only"] = args.goldenrun_only
if parguments["goldenrun_only"]:
            parguments["goldenrun"] = True
```